argocd/local.argocd.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+
+metadata:
+  name: local-application
+  namespace: argocd
+
+spec:
+  project: default
+
+  destination:
+    namespace: local
+    server: https://kubernetes.default.svc
+
+  source:
+    repoURL: https://github.com/theelasticband/devops/
+    targetRevision: klincharov
```

Comment:
targetRevision is set to branch klincharov for local testing of branch, for the sake of completeness. 
Value should be changed to HEAD before merge to main.